lend/src/components/Common/toasts.tsx

```diff
@@ -8,6 +8,63 @@ import Toast from ".//BaseToast";
 import { formatBigNumber } from "../../utils/math";
 import { BigNumber } from "ethers";
 import { ReferralContext } from "../../hooks/referralContext";
+import { SecondaryText } from "shared/lib/designSystem";
+import { useEffect } from "react";
+
+const getReferralText = (
+  type: "Referee" | "Referrer",
+  amount: number,
+  referralCode?: string
+) => {
+  switch (type) {
+    case "Referee":
+      return (
+        <span>
+          <SecondaryText fontSize={12} lineHeight={16} className="mt-1">
+            You earned
+          </SecondaryText>
+          <SecondaryText
+            fontSize={12}
+            lineHeight={16}
+            color="#FFFFFF"
+            className="mt-1"
+          >
+            {` ${amount} RBN `}
+          </SecondaryText>
+          <SecondaryText fontSize={12} lineHeight={16} className="mt-1">
+            from using referral code
+          </SecondaryText>
+          <SecondaryText
+            fontSize={12}
+            lineHeight={16}
+            color="#FFFFFF"
+            className="mt-1"
+          >
+            {` ${referralCode}`}
+          </SecondaryText>
+        </span>
+      );
+    case "Referrer":
+      return (
+        <span>
+          <SecondaryText fontSize={12} lineHeight={16} className="mt-1">
+            You earned
+          </SecondaryText>
+          <SecondaryText
+            fontSize={12}
+            lineHeight={16}
+            color="#FFFFFF"
```

Comment:
can set this color to colors.primaryText instead

Reply:
Fixed in [dedc273](https://github.com/ribbon-finance/ribbon-frontend/pull/624/commits/dedc2738370b87aa5d70a8894eb84e998b852743)